app/helpers/stats_helper.rb

```diff
@@ -6,7 +6,8 @@ def options_for_department_select(departments)
 
   def options_for_organisation_select(department)
     default_option = [["Sélection", [["Toutes les organisations", "0"]]]]
-    grouped_organisations = department.organisations
+    grouped_organisations = department.organisations.where.not(id: ENV.fetch("ORGANISATION_IDS_WHERE_STATS_DISABLED",
```

Comment:
nitpick: on pourrait déplacer `ENV.fetch("ORGANISATION_IDS_WHERE_STATS_DISABLED", "").split(",")` dans une méthode `disable_organisation_ids` et utiliser `reject`  plutôt que `where.not` pour ne pas faire de query sql ici: 
` grouped_organisations = department.organisations.reject { |o| o.id.in?(disabled_organisation_ids) }`

Reply:
Done here https://github.com/gip-inclusion/rdv-insertion/pull/2524/commits/547845bdf2c9b65a575c55e4343b1cf29e3db88a